Sprint-2/debug/recipe.js

```diff
@@ -12,4 +14,4 @@ const recipe = {
 
 console.log(`${recipe.title} serves ${recipe.serves}
   ingredients:
-${recipe}`);
+${recipe.ingredients}`);
```

Comment:
This is good and it works as well. Can you please think of accessing each item & format as below?

bruschetta serves 2 
ingredients:
olive oil
tomatoes
salt
pepper